### PR TITLE
adding new meeting on mpl calendar

### PR DESCRIPTION
**Pull request by noatamir**
👋 Please let me know if this looks good or if I should change anything.
We might need to make changes to the dates moving forward so I need to learn how to get this right 😉

### CODE REVIEW

File: calendars/matplotlib.yaml [hunk 1]
```diff
@@ -17,3 +17,22 @@ events:
         # seconds, minutes, hours, days, weeks, months, years
         days: 7
       until: 2025-01-01 00:00:00
+
+timezone: Europe/Berlin
+events:
```
Comment (tupui) [inline calendars/matplotlib.yaml:22]: These 2 are not necessary
```suggestion
```

File: calendars/matplotlib.yaml [hunk 2]
```diff
@@ -17,3 +17,22 @@ events:
...
+      Meeting notes: https://hackmd.io/@matplotlib
+
+    begin: 2022-05-10 19:00:00
+    end: 2022-05-10 20:00:00
+    url: https://us06web.zoom.us/j/81285851006?pwd=Tks2QjRkNWh5NGw0TmU1RUwwOVluZz09
+    repeat:
+      interval:
+        # seconds, minutes, hours, days, weeks, months, years
+        days: 28
+      until: 2025-01-01 00:00:00 +00:00
```
Comment (tupui) [inline calendars/matplotlib.yaml:38]:
```suggestion
      until: 2025-01-01 00:00:00 +00:00

```

**Review by tupui — CHANGES_REQUESTED** [review]
Thank you @noatamir! I think it should work after my suggestion. FYI, when the CI is green, we can actually check the calendar in the preview.

**Comment by noatamir** [comment]
had some back & forth with my terminal but I was just repeating the same commit. you can delete the repeated ones if needed 😇

File: calendars/matplotlib.yaml [hunk 3]
```diff
@@ -17,3 +17,21 @@ events:
...
```
Comment (tupui) [inline calendars/matplotlib.yaml:36]: Maybe this is the issue
```suggestion
      until: 2025-01-01 00:00:00
```

**Review by tupui — COMMENTED** [review]
No worries about the commits as we are only squashing on this repo.

File: calendars/matplotlib.yaml (hunk 3 above)
Comment (melissawm) [reply calendars/matplotlib.yaml:36]: I was going to say this is what we have for the NumPy meetings, but indeed now there is a timezone option which is maybe conflicting with this note here

Comment (tupui) [reply calendars/matplotlib.yaml:36]: If we want multiple time zone support that's something we could add in our lib.

**Review by tupui — APPROVED** [review]
LGTM now

**Comment by tupui** [comment]
Oh actually I don't have credentials on this repo 😅 @jarrodmillman can you merge this?

**Review by jarrodmillman — APPROVED** [review]

**Comment by noatamir** [comment]
Thank you! 🙇‍♀️

**Pull request merged**